projects/joystick-servo-gimbal.md

```diff
@@ -1 +1,84 @@
-# Arduino Joystick and Servo Gimbal
\ No newline at end of file
+# Candy Dispenser
```

Comment:
shouldn't the title be `# Arduino Joystick and Servo Gimbal` instead of `# Candy Dispenser`?